### PR TITLE
Feat cart

### PR DESCRIPTION
**Pull request by JowelTisso**

### CODE REVIEW

**Review by SukanyaSen530 — COMMENTED** [review]
1. Add the items' count on navbar
2.  On page refresh, in product's page, the original state changes back but the checkbox still shows checked.

**Comment by JowelTisso** [comment]
> 1. Add the items' count on navbar
> 2. On page refresh, in product's page, the original state changes back but the checkbox still shows checked.

Thanks for the review, both the issue has been fixed

**Review by SukanyaSen530 — APPROVED** [review]

**Comment by narharikale** [comment]
clean code theme is loolking awesome 🤩

File: src/context/reducer/cartReducerHelper.js [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+      };
+    case REMOVE_FROM_CART:
+      console.log(state.totalItems + "-" + action.payload.quantity);
+      console.log(
+        state.totalPrice +
+          "-" +
+          parseFloat(action.payload.cartItem.price) +
+          "*" +
+          action.payload.quantity
+      );
```
Comment (manismk) [inline src/context/reducer/cartReducerHelper.js:82]: can remove the console log

**Review by narharikale — APPROVED** [review]
good use of constants , clean code
theme is loolking awesome 🤩

File: src/context/reducer/cartReducerHelper.js [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+      totalItems: state.totalItems + 1,
+      totalPrice: state.totalPrice + parseFloat(action.payload.cartItem.price),
+      cartItems: [...state.cartItems],
+    };
+  }
+  return { ...state };
+};
+
+export const reducer = (state, action) => {
+  console.log("reducer");
```
Comment (manismk) [inline src/context/reducer/cartReducerHelper.js:65]: can remove the console log

**Review by manismk — APPROVED** [review]
Good naming convention and file structure.
Liked the way you created helper functions for the reducers

File: src/context/reducer/cartReducerHelper.js (hunk 1 above)
Comment (JowelTisso) [reply src/context/reducer/cartReducerHelper.js:82]: ok, got it

File: src/context/reducer/cartReducerHelper.js (hunk 2 above)
Comment (JowelTisso) [reply src/context/reducer/cartReducerHelper.js:65]: done

**Pull request merged**